### ISSUE TITLE
Problem with MID

### ISSUE THREAD

**Issue by mcasto** [opis/database#84]
When I enter this query (which, as near as I can tell from the documentation is correct):

```
$db->from('events')
    ->where(function ($e) {
      $e->mid('eventDatetime', 1, 10);
    }, true)
    ->is('2021-04-11')
    ->select()
    ->all();
```

I get an error: `SELECT * FROM events WHERE MID(eventDatetime, 110) = '2021-04-11'`

It looks like the parsing of the MID somehow loses the comma between 1 & 10.

I had the same problem with code directly from the documentation (with minor tweaks for my data):
```
    $db->from('events')
    ->select(function ($include) {
      $include->mid('eventDateTime', 1, null, 10);
    })
    ->all();
```

Error: `SELECT MID(eventDateTime, 110) FROM events`

**Comment by sorinsarca** [comment]
I found the problem, we'll add a fix soon

**Comment by msarca** [comment]
This should be fixed in v4.2.1.

**Comment by mcasto** [comment]
Awesome! Works like charm! Thanks for the quick update.
